app/src/main/res/layout/fragment_add_title_descriptions.xml

```diff
@@ -50,16 +50,16 @@
 
 
             <ImageView
-                android:id="@+id/arrows"
-                android:layout_width="40dp"
-                android:layout_height="match_parent"
+                android:id="@+id/arrow"
+                android:layout_width="16dp"
+                android:layout_height="8dp"
```

Comment:
Would it be better if it has a bigger clickable area? (by adding paddings?)

Reply:
Good catch! Done!